src/python/pants/notes/1.25.x.rst

```diff
@@ -10,6 +10,46 @@ The ``1.25.x`` series brings two major changes to Pants:
 
 Please see https://groups.google.com/forum/#!topic/pants-devel/3nmdSeyvwU0 for more information.
 
+1.25.1rc0 (03/20/2020)
+----------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Zinc: Use an Array[PublishDiagnosticsParam] for LSP compliance (#9292)
+  `PR #9292 <https://github.com/pantsbuild/pants/pull/9292>`_
+
+* Enable zinc to log diagnostics for jvm languages (#9228)
+  `PR #9228 <https://github.com/pantsbuild/pants/pull/9228>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Report warnings and errors to reporting server for JVM targets (#9293)
+  `PR #9293 <https://github.com/pantsbuild/pants/pull/9293>`_
+
+Bugfixes
+~~~~~~~~
+
+* Fix relative path in Zinc compiler (#9261)
+  `PR #9261 <https://github.com/pantsbuild/pants/pull/9261>`_
+
+* Revert "Load the bootstrapped zinc compiler from the zinc server's classpath. (#8753)" (#9226)
+  `PR #9226 <https://github.com/pantsbuild/pants/pull/9226>`_
+
+* fixing scoverage sourcePath to be relative in OSS (#8132)
+  `PR #8132 <https://github.com/pantsbuild/pants/pull/8132>`_
+
+Refactoring, Improvements, and Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Fix `PathGlobs` to be deterministic for more cache hits (#9347)
+  `Issue #9346 <https://github.com/pantsbuild/pants/issues/9346>`_
```

Comment:
Oh, ditto on this. I think it works to keep the Issue link, but I usually delete it. I reason that If they care enough about this change, they will go to the PR link and then be referred to the issue link for more context.

Reply:
OK. Makes sense. I think for now I'll leave it to avoid having to restart CI from the beginning (Sorry: just saw this comment now). But will keep in mind for next time.

Reply:
Ah, so with release notes, we make an exception that after a valid CI run (modulo the expected failures), you're allowed to make changes and not re-run CI since these are doc changes :)

So, once CI goes ~green on this one, you can still make this change, push it, and immediately merge.